pkg/apis/garden/v1beta1/defaults.go

```diff
@@ -241,6 +185,33 @@ func SetDefaults_Seed(obj *Seed) {
 	if obj.Spec.Protected == nil {
 		obj.Spec.Protected = &falseVar
 	}
+
+	var (
+		defaultPodCIDR             = DefaultPodNetworkCIDR
+		defaultServiceCIDR         = DefaultServiceNetworkCIDR
+		defaultPodCIDRAlicloud     = DefaultPodNetworkCIDRAlicloud
+		defaultServiceCIDRAlicloud = DefaultServiceNetworkCIDRAlicloud
+	)
+
+	if obj.Spec.Networks.ShootDefaults == nil {
+		obj.Spec.Networks.ShootDefaults = &ShootNetworks{}
+	}
+
+	if v, ok := obj.Annotations[garden.MigrationSeedProviderType]; ok && v == "alicloud" {
+		if obj.Spec.Networks.ShootDefaults.Pods == nil {
+			obj.Spec.Networks.ShootDefaults.Pods = &defaultPodCIDR
+		}
+		if obj.Spec.Networks.ShootDefaults.Services == nil {
+			obj.Spec.Networks.ShootDefaults.Services = &defaultServiceCIDR
+		}
+	} else {
```

Comment:
Please check that you assign the `defaultPodCIDR ` and `defaultServiceCIDR` here.